Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+import SwiftUI
+
+public typealias SliderValue = BinaryFloatingPoint & Strideable & CustomStringConvertible
+
+/// Enables selection from a range of values.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/slider/)
+public struct Slider<Value>: View where Value: SliderValue, Value.Stride: BinaryFloatingPoint {
+
+    enum SelectedValue {
+        case single(Binding<Value>?)
+        case range(Binding<ClosedRange<Value>>?)
+    }
+
+    @Environment(\.sizeCategory) var sizeCategory
```

Comment:
The sizeCategory has some glitches. Not only when the slider causes the surrounding context to be changed as a result, but even in the initialize state

https://user-images.githubusercontent.com/35844477/235114460-0c0212a5-2dee-48a1-b643-d3d430bc8e86.mov